_scripts/instruction-widget/templates/getting-started/renewal.html

```diff
@@ -18,16 +18,11 @@ <h3>Automating renewal</h3>
 </pre>
 
 If that appears to be working correctly, you can arrange for automatic renewal
-by adding a <tt>cron</tt> or <tt>systemd</tt> job which runs the
+by adding a <tt><a href="http://www.unixgeeks.org/security/newbie/unix/cron-1.html">cron</a></tt> or <tt><a href="https://en.wikipedia.org/wiki/Systemd">systemd</a></tt> job which runs the
```

Comment:
Two things:

1. Can we change the text of this line to say "by adding a cron job or systemd timer which..." (with "systemd timer" being the link text)? This to me helps clarify what to look for if you go with systemd.
2. Can we use https://wiki.archlinux.org/index.php/Systemd/Timers for the link about systemd timers? I think it provides a pretty good distro agnostic description of the subject.